torch/onnx/_internal/exporter.py

```diff
@@ -1190,16 +1190,21 @@ def __init__(
             self._assert_fake_tensor_mode()
 
     def export(self) -> ONNXProgram:
+        from torch.export._trace import DEFAULT_EXPORT_DYNAMO_CONFIG
```

Comment:
nit: top level import

Reply:
need to be here to prevent circular dependency =/